tasks/zadanie_23.php

```diff
@@ -29,3 +29,26 @@
 */
 
 $postcodes = $params[0]; // tej linijki nie ruszamy :)
+
+function checkPostcode( string $check) : bool
```

Comment:
Tutaj bardzie adekwatną nazwą dla zmiennej wydaje mi się że byłoby $postcode ;) Poza tym spoko ;)
```suggestion
function checkPostcode(string $postcode) : bool
```